lib/cgi/escape.rb

```diff
@@ -219,10 +225,8 @@ def unescapeElement(string, *elements)
     end
   end
 
-  # Synonym for CGI.escapeElement(str)
```

Comment:
The "It is an alias for ..." is now rendered by RDoc by default, repeating it looked weird:

<img width="451" height="163" alt="Image" src="https://github.com/user-attachments/assets/ace5b16c-7ca7-4faf-9c5f-56b3427476f8" />